src/frontend/components/horizontalCard/horizontal-card.js

```diff
@@ -1,11 +1,15 @@
 import { useCart } from "../../contexts/cart-context";
+import { useWishlist } from "../../contexts/wishlist-context";
 import "./horizontal-card.css";
 const laptopImage = require('../../assets/laptop.png');
 
 
 const HorizontalCard = (product)=>{
   
-    const {removeFromCart}= useCart()
+    const {removeFromCart , moveToWishlist}= useCart()
+    const { addToWishList } = useWishlist();
+    
+    
     const incrementQuantity=(product)=>{
         // {cart.map((item)=>item._id===product._id) ? {...item, quantity:product.quantity+1} :item
```

Comment:
remove commented code if not required

Reply:
yeah that one is required will work on that